requirements-frozen.txt

```diff
@@ -53,7 +53,7 @@ google-crc32c==1.3.0
 google-resumable-media==2.3.3
 googleapis-common-protos==1.56.4
 grpc-google-iam-v1==0.12.4
-grpcio==1.48.1
+grpcio==1.56.0
 grpcio-status==1.48.1
```

Comment:
Should we upgrade `grpcio-status` to 1.56.0 as well in this PR or separately?

Reply:
yeah usually they're supposed to match -- I'm not surprised dependabot didn't do this properly